src/views/FAQ.vue

```diff
@@ -15,9 +15,8 @@
         </v-expansion-panels>
 
         <p class="mt-8">
-          Weitere Informationen findest du in unserem
-          <base-link href="https://clockgu.gitbook.io/clock-benutzerhandbuch"
-            >Benutzerhandbuch</base-link
+          {{ $t("faq.text") }}
```

Comment:
Das erinnert mich an diese Stelle im [DashboardWelcome.vue](https://github.com/ClockGU/clock-frontend/blob/ebe0419f5bd7bc95593cf31ca51c41d679a850b4/src/components/dashboard/DashboardWelcome.vue#L4)

Diese Variante ist insofern etwas eleganter als der Link in die Übersetzung eingefügt wird und der komplette Satz (inkl. Punkt am Ende 😁 ) in der locale steht. Aber ich bin nicht sicher, ob das ansonsten Vorteile hat oder äquivalent ist - ist ja auch sehr ähnlich.

Reply:
Allerdings würde ich hier die Formulierung so wählen

`Please refer to the user manual for further information.`

und dann wäre der Link in der Mitte des Satzes und der Punkt kann nicht hardcoded sein. Ich denke, ich würde diese Variante bevorzugen. Ob das mit `<template #link>` sein muss oder einfacher geht, weiß ich gerade nicht; ggf. kannst Du auch einen Blick in die Doku zu [vue-i18n](https://vue-i18n.intlify.dev/introduction.html) werfen.

Reply:
Ich pushe mal eine kleine Änderung (ich finde `url` nämlich besser als link in DashboardWelcome). Aber den Text anpassen müsstest Du dann @SoerenGrieger, wenn fertig.

Reply:
Hab ich angepasst, ist wesentlich eleganter mit dem `base link`

Reply:
Schon gesehen danke! 

**resolved**